src/dao/pages/ProposalCard.tsx

```diff
@@ -134,7 +135,11 @@ export const ProposalCard = (props: ProposalCardProps) => {
                     <HederaIcon />
                     <SendTokenIcon boxSize={5} stroke={Color.Destructive._400} marginRight={1} marginLeft={2} />
                     <Text textStyle="p medium regular" color={Color.Neutral._900} textAlign="start">
-                      {formatTokenAmountWithDecimal(amount, Number(token?.data.decimals ?? 0))} {tokenSymbol}
+                      {isFungible(token?.data.type) &&
+                        formatTokenAmountWithDecimal(amount, Number(token?.data.decimals ?? 0)) + " "}
+                      {token?.data.symbol}
```

Comment:
See a similar comment in the `ProposalActionDetails.tsx`  file.

Reply:
Done.